pnpm-lock.yaml

```diff
@@ -16741,7 +16741,7 @@ snapshots:
   '@babel/plugin-syntax-json-strings@7.8.3(@babel/core@7.26.10)':
     dependencies:
       '@babel/core': 7.26.10
-      '@babel/helper-plugin-utils': 7.26.5
+      '@babel/helper-plugin-utils': 7.24.0
```

Comment:
This change should not be here, try to remove it, cleaning up your node_modules and reinstalling, this should disappear.